src/components/StorageOverview/Capacity/Capacity.js

```diff
@@ -9,48 +11,131 @@ import {
   DashboardCardHeader,
   DashboardCardTitle,
 } from '../../Dashboard/DashboardCard';
+import { Dropdown } from '../../Form/Dropdown';
+import { DashboardCardActionsBody } from '../../Dashboard/DashboardCard/DashboardCardActionsBody';
+
 import { StorageOverviewContext } from '../StorageOverviewContext';
 import { CapacityItem } from '../../Dashboard/Capacity/CapacityItem';
 import { formatBytes } from '../../../utils';
 import { getCapacityStats } from '../../../selectors';
 import { CapacityBody } from '../../Dashboard/Capacity/CapacityBody';
+import { TOTAL, CAPACITY, VMSVSPODS } from './strings';
+
+const capacityTypes = [TOTAL, CAPACITY, VMSVSPODS];
+
+export const CapacityImplementation = ({ Total, Used, title, LoadingComponent }) => (
```

Comment:
why having a separate component ? Can't you just have the content of `CapacityImplementation` directly in `DashboardCardBody` ?

Reply:
I can do that, was trying to keep it consistent with the way the other dropdowns are implemented. As metrics are sent to a `...Body` component which renders the card.
Couldn't reuse `CapacityBody` so renamed it to `CapacityImplementation`.

Reply:
I still think it would be cleaner to have content directly in `DashboardCardBody` - you are not doing any other calculations but others (ie TopConsumers) do. So it's just unnecessary wrapping.